src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -166,15 +166,12 @@
 
     // text area from getting updated. This is used in cases where the source
     // couldn't be parsed, and the user is given the option to edit it.
-    private boolean lastSourceAccepted = true; // This indicates whether the last
-
-    // attempt
+    private boolean lastFieldAccepted = true;
+    private boolean lastSourceAccepted = true; // This indicates whether the last attempt
     // at parsing the source was successful. It is used to determine whether the
```

Comment:
At first sight it's quite obscure to which field these comment belongs.
Can you please move them (and the other comments) before their corresponding field and use JavaDoc?